File: python/random_passphrase.py
import secrets
import sys

def passphrase(length):
  words = open(sys.path[0] + '/dictionary.txt').read().split("\n")
  output = []
  while len(output) < length:
    word = secrets.choice(words)
    if word not in output: output.append(word)
  return ' '.join(output)

print(passphrase(3))

File: ruby/random_passphrase.rb
def passphrase(length)
  words = File.read(__dir__ + '/dictionary.txt').lines.map(&:split)
  output = []
  while output.length < length
    word = words[rand(words.length)]
    output.push(word) unless output.include? word
  end
  output.join(' ')
end

puts passphrase(3)
